release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3334,6 +3334,8 @@ For more information see, xref:../networking/ptp/configuring-ptp.adoc#configurin
 
 * Previously, when you deployed IPv6-only hosts from a dual-stack cluster, an issue prevented the Baseboard Management Controller (BMC) from receiving the correct callback URL. Instead, the BMC received an IPv4 URL. With this update, the issue no longer occurs because the IP version of the URL depends on the IP version of the BMC address.(link:https://issues.redhat.com/browse/OCPBUGS-23903[*OCPBUGS-23903*])
 
+* Previously, on an {sno-caps} that has guaranteed CPUs and where Interrupt Request (IRQ) load balancing is disabled, large latency spikes could occur at container startup. With this update, the issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-22901[*OCPBUGS-22901*])
```

Comment:
https://issues.redhat.com/browse/OCPBUGS-24281 I think is the bug that is shipped with 4.14.z.

Reply:
sorry not sure what you want done here @JoeAldinger, this bug OCPBUGS-22901 is described in bugs section so I want to record that as fixed in this z-stream. The JIRA asked me to add this do you want me to add [OCPBUGS-24281](https://issues.redhat.com/browse/OCPBUGS-24281) this in brackets here too ?

Reply:
The bug you referenced is marked fixed in 4.15.0. Engr cloned it 24281 for the zstream and that is what the advisory picked up and attached to the advisory referenced in this z-stream. In short, the bug reference needs to be changed or at least add that cloned bug please.

Reply:
Thanks for digging that out I will add the cloned bug what is the best practice for that just add link:https://issues.redhat.com/browse/OCPBUGS-24181[*OCPBUGS-24181*])

Reply:
Yeah, you can just add the link because the bug already shipped with that zstream.

Reply:
@mburke5678 that bug needs added.

> [issues.redhat.com/browse/OCPBUGS-24281](https://issues.redhat.com/browse/OCPBUGS-24281) I think is the bug that is shipped with 4.14.z.